templates/extra/faq.html

```diff
@@ -1,176 +1,67 @@
-        </ul>
-        <ul class="vertical menu" data-accordion-menu>
-            <li>
-                <a href="#">
-                    <h5>Once an artist gets funded, how do they get money?</h5>
-                </a>
-                <ul class="menu vertical nested">
-                    <li>
-                        <a href="#">
-                            <p>
-                                We send the artist a check for the full amount they raised or transfer via PayPal.
-                            </p>
-                        </a>
-                    </li>
-                </ul>
-            </li>
-        </ul>
+    <h5 class="color-blue">So do I own equity in the artists?</h5>
+    <p>
+        No, you don't own any equity in the artist or their music. The artist sets a percentage they want to give back to the investors based on revenue for that project. Since PerDiem distributes the music, it’s easy to maintain integrity between the artist and investors.
+    </p>
+    <div class="break">
+        <hr>
+    </div>
+    <h5 class="color-blue">Once I invest, how do I get money?</h5>
+    <p>
+        With any investment, it may take a while for you to see a return on your investment. When you decide to sell your shares, we purchase them back from you at their current value which may be more or less than you purchased them at. Simply “Request Cash Out” on your user profile page and we will send your earnings via PayPal in 1-2 business days.
+    </p>
+    <div class="break">
+        <hr>
+    </div>
+    <h5 class="color-blue">Does the artist own their music?</h5>
+    <p>
+        Yes. Since the band retains full ownership of their music, and makes the choice to "give back" to their fans, there is no need to have contracts between the artists and the fans. You are simply purchasing a record (or multiple records) from the artist and the artist is rewarding you for doing so. PerDiem has exclusive distribution to the music which helps ensure the investors get a direct return on their investment.
+    </p>
+    <div class="break">
+        <hr>
+    </div>
+    <h5 class="color-blue">What happens if an artist leaves the platform?</h5>
+    <p>
+        The system is designed to protect both the artists and the fans. If an artist decides to leave the platform we highly encourage them to do it in a way that doesn't upset or disappoint their investors. In the past, some artists that were unable to fulfill their campaign goals have refunded the money to investors, while others have not. Investing in music is high risk and we encourage people to only invest what they are willing to lose.
+    </p>
+    <div class="break">
+        <hr>
+    </div>
+    <h5 class="color-blue">How does PerDiem make money?</h5>
+    <p>
+        100% of the money from shares purchased goes to the artists, and 100% of revenue is split between the artist and fans. The only fee for the investor is the credit card processing fee + 12.9% + $0.30.
```

Comment:
Is the 12.9% correct? That sounds very high.

Reply:
No, I wrote that line incorrectly. 

I mentioned it on slack a little while ago, but I want to change the $1 fee to 10%. I tried updating it on the backend but ran into some issues. I'll shoot you a message on slack about.